src/hera.cpp

```diff
@@ -326,7 +326,13 @@ void execute(
 
   Name main = Name("main");
   LiteralList args;
-  instance.callExport(main, args);
+
+  try {
+    instance.callExport(main, args);
+  } catch (EndExecution const&) {
```

Comment:
This will not work because the exception type is `EndExecution*` (no match). Must be fixed in other place.

Reply:
Fixed both.